styles.css

```diff
@@ -0,0 +1,271 @@
+body {
+  display: grid;
+  grid-template-areas:
+    "head head"
+    "nav main"
+    "nav main";
+  grid-template-rows: 95px 1fr;
+  grid-template-columns: 230px 1fr;
+  overflow-x: hidden;
+}
+
+body > header {
+  grid-area: head;
+  display: flex;
+  justify-content: space-between;
+  padding: 0 5% 0 17%;
+  min-width: 90%;
+  position: relative;
+  gap: 3rem;
+  align-items: center;
+  background-color: var(--white-color);
+}
+
+body > nav {
+  grid-area: nav;
+  height: 100%;
+  width: 15%;
+  position: fixed;
+  z-index: 1;
+  top: 0;
+  left: 0;
+  padding: 1.3rem 1rem 0 2rem;
+  background-color: #ffffff;
+}
+
+body > main {
+  grid-area: main;
+  border-radius: 1.5rem 0 0 0;
+  padding: 2rem 10% 0;
```

Comment:
this can be made consistent without mixing rem and % . 1 unit can be used for consistency